server/http_integration_test.go

```diff
@@ -458,6 +465,117 @@ func TestHTTPServer_XFHHeader(t *testing.T) {
 	cleanup(shutdown, t)
 }
 
+func TestHTTPServer_Gzip(t *testing.T) {
```

Comment:
This integration test does not cover a missing AE header.

Reply:
A Missing AE header is covered by the netx test `TestServer_DisableCompression`

Reply:
`TestServer_DisableCompression` tests the proxy (backend) and not the file-handler which is called within this test. So ensuring a raw file serving without AE header would complete this case here.

Reply:
A lot of raw file serving tests ist tested by `TestHTTPServer_ServeHTTP` in the same test file.

Reply:
So changing other tests and implementations still ensures the expected outcome for this gzip feature?
Please modify the test to (file)serve and verify a compressed and uncompressed version depending on the clients AE header. This is the expectation within this context here.